src/rpc/ngHandlers/AccountInfo.cpp

```diff
@@ -0,0 +1,145 @@
+
+    auto const range = sharedPtrBackend_->fetchLedgerRange();
+    auto const lgrInfoOrStatus = RPC::getLedgerInfoFromHashOrSeq(
+        *sharedPtrBackend_,
+        yield,
+        input.ledgerHash,
+        input.ledgerIndex,
+        range->maxSequence);
+
+    if (auto const status = std::get_if<RPC::Status>(&lgrInfoOrStatus))
+        return Error{*status};
+
+    auto const lgrInfo = std::get<ripple::LedgerInfo>(lgrInfoOrStatus);
+
+    auto const accountStr = input.account.value_or(input.ident.value_or(""));
+    auto const accountID = RPC::accountFromStringStrict(accountStr);
+    auto const accountKeylet = ripple::keylet::account(*accountID);
+    auto const accountLedgerObject = sharedPtrBackend_->fetchLedgerObject(
+        accountKeylet.key, lgrInfo.seq, yield);
+    if (!accountLedgerObject)
+        return Error{RPC::Status{
+            RPC::RippledError::rpcACT_NOT_FOUND, "accountNotFound"}};
+
+    ripple::STLedgerEntry const sle{
+        ripple::SerialIter{
+            accountLedgerObject->data(), accountLedgerObject->size()},
+        accountKeylet.key};
+    if (!accountKeylet.check(sle))
+        return Error{RPC::Status{RPC::RippledError::rpcDB_DESERIALIZATION}};
+    // Return SignerList(s) if that is requested.
+    if (input.signerLists)
+    {
+        // We put the SignerList in an array because of an anticipated
+        // future when we support multiple signer lists on one account.
+        auto const signersKey = ripple::keylet::signers(*accountID);
+        // This code will need to be revisited if in the future we
+        // support multiple SignerLists on one account.
+        auto const signers = sharedPtrBackend_->fetchLedgerObject(
+            signersKey.key, lgrInfo.seq, yield);
+        std::vector<ripple::STLedgerEntry> signerList;
+        if (signers)
+        {
+            ripple::STLedgerEntry const sleSigners{
+                ripple::SerialIter{signers->data(), signers->size()},
+                signersKey.key};
+            if (!signersKey.check(sleSigners))
+                return Error{
+                    RPC::Status{RPC::RippledError::rpcDB_DESERIALIZATION}};
+
+            signerList.push_back(sleSigners);
+        }
+        return Output(
+            lgrInfo.seq, ripple::strHex(lgrInfo.hash), sle, signerList);
+    }
+    return Output(lgrInfo.seq, ripple::strHex(lgrInfo.hash), sle);
+}
+
+void
+tag_invoke(
+    boost::json::value_from_tag,
+    boost::json::value& jv,
+    AccountInfoHandler::Output const& output)
+{
+    jv = boost::json::object{
+        {JS(account_data), RPC::toJson(output.accountData)},
+        {JS(ledger_hash), output.ledgerHash},
+        {JS(ledger_index), output.ledgerIndex}};
+    if (output.signerLists)
+    {
+        jv.as_object()[JS(signer_lists)] = boost::json::array();
+        for (auto const& signerList : output.signerLists.value())
+            jv.as_object()[JS(signer_lists)].as_array().push_back(
+                RPC::toJson(signerList));
```

Comment:
Let's add a todo for later here to 
- replace `toJson` with `tag_invoke`
- not look up `signer_lists` + `as_array` each iteration
This really should be just `jv.as_object().insert_or_assign(JS(signer_lists)), output.signerLists);` if all `tag_invoke`s are correctly added.

Reply:
I'm ok with that. Just add a TODO comment about this desired change then and let's move on :)